src/main/java/org/folio/innreach/domain/service/impl/ContributionServiceImpl.java

```diff
@@ -0,0 +1,34 @@
+package org.folio.innreach.domain.service.impl;
+
+import lombok.RequiredArgsConstructor;
+import org.folio.innreach.client.InventoryClient;
+import org.folio.innreach.domain.entity.Contribution;
+import org.folio.innreach.domain.service.ContributionService;
+import org.folio.innreach.repository.ContributionRepository;
+import org.springframework.stereotype.Service;
+
+@RequiredArgsConstructor
+@Service
+public class ContributionServiceImpl implements ContributionService {
+
+  private final ContributionRepository repository;
+  private final InventoryClient client;
+
+  @Override
+  public void startInitialContribution() {
+    var contribution = emptyContribution();
+    repository.save(contribution);
+    client.startInitialContribution();
+  }
+
+  private Contribution emptyContribution() {
+    var contribution = new Contribution();
+    contribution.setStatus(Contribution.Status.IN_PROGRESS);
+    contribution.setRecordsTotal(0L);
```

Comment:
@maryna-steshenko @dmtkachenko Shouldn't this property be populated with value from mod-inventory-storage?

Reply:
It should be though I cannot say exactely how to do it.

Reply:
Yeah.. me too. I guess we can implement it later